index.js

```diff
@@ -14,7 +14,7 @@ let next=new Date();
 let passthrough = false;
 let officers = JSON.parse(process.env.OFFICERS);
 
-const client = new discord.Client({autoReconnect: true});
```

Comment:
autoReconnect is not a field in ClientOptions. It is implied it should automatically reconnect.